Podcast/RecommendedEpisodesOuterTableViewCell.swift

```diff
@@ -114,7 +114,7 @@ class RecommendedEpisodesOuterTableViewCell: UITableViewCell, UITableViewDelegat
         guard let indexPath = currentlyPlayingIndexPath, let previousEpisode = dataSource?.recommendedEpisodesTableViewCell(cell: self, dataForItemAt: indexPath) else { return }
         // check if previously playing episode is no longer playing
         if previousEpisode.id != episode?.id {
-            (tableView.cellForRow(at: indexPath) as? EpisodeTableViewCell)?.episodeUtilityButtonBarView.setPlayButtonToState(isPlaying: false)
+            (tableView.cellForRow(at: indexPath) as? EpisodeTableViewCell)?.episodeSubjectView.episodeUtilityButtonBarView.setPlayButtonToState(isPlaying: false)
```

Comment:
We should probably also think about abstracting out the button state methods. So that I can just tell the cell to be playing or not, in case we have different playing buttons/views. Then all our cells could implement some interface for our user actions like playing, liking, bookmarking, etc. so our cell calls are consistent.

Reply:
I made this change for now. We have the same code repeated for many of our delegation methods which I plan to refactor soon. I won't include it in this PR because I want us all to discuss first.